backend/typescript/models/waiver.model.ts

```diff
@@ -0,0 +1,39 @@
+import mongoose, { Schema, Document } from "mongoose";
+
+export interface Waiver extends Document {
+  a: string;
+  b: string;
+  c: string;
+  d: string;
+  e: string;
+  f: string;
+}
+
+const WaiverSchema: Schema = new Schema({
+  a: {
+    type: String,
+    required: true,
+  },
+  b: {
+    type: String,
+    required: true,
+  },
+  c: {
+    type: String,
+    required: true,
+  },
+  d: {
+    type: String,
+    required: true,
+  },
+  e: {
+    type: String,
+    required: false,
+  },
+  f: {
+    type: String,
+    required: false,
+  },
```

Comment:
hmm this works, but I think it might be better to change it so the number of clauses in the waiver is dynamic - we could store the clauses in a string array, for example. we just don't want to restrict the waiver to a strict number of fields in case that's subject to change. also, there should be some way of marking each clause as optional. let me know if you have any qs on this